.circleci/config.yml

```diff
@@ -0,0 +1,48 @@
+aliases:
+  - &downgrade-cocoapods
+    name: Downgrade to CocoaPods 1.4
+    command:  |
+      echo Y | sudo gem uninstall cocoapods
+      sudo gem install cocoapods -v 1.4
+    when: always
+
+  - &generate-workspace
+    name: Setup
+    command: node ./prepareios.js
+    when: always
+
+mac: &mac
+  working_directory: ~/SalesforceMobileSDK-ReactNative/ios/SalesforceReactTests
+  macos:
+    xcode: "9.4.1"
```

Comment:
Shouldn't this be `Xcode 10` now?

Reply:
Probably - this is just a first cut (basically a copy/paste job on https://github.com/forcedotcom/SalesforceMobileSDK-Package/blob/master/.circleci/config.yml). I just want to see it running. Then I will refine.